_includes/codepens/advcode/index.html

```diff
@@ -0,0 +1,41 @@
+<table>
+  <thead>
+      <tr>
+          <th style="width: 50%"><h2>The Code Plugin</h2></th>
+          <th style="width: 50%"><h2>The Advanced Code Editor Plugin</h2></th>
+      </tr>
+  </thead>
+  <tbody>
+      <tr>
+          <td style="width: 50%">
+            <textarea class="codedemo">
+              <p>The code (<code>code</code>) plugin provides a dialog for viewing and editing the HTML of the editor content.</p>
```

Comment:
Should the first "code" be `Code` here, as it's the name of the plugin?

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1594/commits/42b7fcf5c90819e237ab7dccd63ff44883971ee0